Code/Framework/AzToolsFramework/AzToolsFramework/Prefab/Overrides/PrefabOverridePublicHandler.cpp

```diff
@@ -51,11 +51,15 @@ namespace AzToolsFramework
             }
         }
 
-        bool PrefabOverridePublicHandler::AreOverridesPresent(AZ::EntityId entityId)
+        bool PrefabOverridePublicHandler::AreOverridesPresent(AZ::EntityId entityId, const AZ::Dom::Path& prefix)
         {
```

Comment:
I feel like we can have a separate function to check if overrides are present for components and properties. I'm okay with both solutions.

Another thought is we can use `relativePathToValue` or something similar rather than `prefix`? I used to find it hard to follow (in some functions in our interfaces) what prefix it is. From my first glance, I thought it was a prefix to an entity.

Reply:
+1, interface seems fine but I'd call out that it's specifically a dom subpath of the entity